Database/Corrections/sodItemFixes.lua

```diff
@@ -32,6 +32,14 @@ function SeasonOfDiscovery:LoadItems()
         [209846] = {
             [itemKeys.objectDrops] = {409692},
         },
+        [210187] = {
+            [itemKeys.name] = "Venture Co. Work Order",
+            [itemKeys.npcDrops] = {3988,4070,3992,3993},
+            [itemKeys.objectDrops] = nil,
+            [itemKeys.itemDrops] = nil,
+            [itemKeys.vendors] = nil,
+            [itemKeys.startQuest] = {78270},
```

Comment:
Only keep this override in there and move the rest back to the base file

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5339/commits/2d618139cf5b19c5a1926c230a55a5dd9dd0360c